src/main/java/org/kainos/ea/core/JobRoleRequestValidator.java

```diff
@@ -0,0 +1,32 @@
+package org.kainos.ea.core;
+
+import org.kainos.ea.cli.JobRoleRequest;
+
+public class JobRoleRequestValidator {
+
+    public String isValidJobRole(JobRoleRequest jobRoleRequest) {
+
+        if(jobRoleRequest.getJobRoleTitle().length() < 5){
+            return "The job role title must be at least 5 characters";
+        }
+
+        if(jobRoleRequest.getJobRoleTitle().length() > 100){
+            return "The job role title must be less than 100 characters";
+        }
+
+        if(jobRoleRequest.getBandId() < 1){
+            return "You must choose a valid band";
```

Comment:
You can go off to the DB here and check that the band you've selected is actually valid